saleor/graphql/webhook/mixins.py

```diff
@@ -0,0 +1,21 @@
+from django.core.exceptions import ValidationError
+
+from saleor.webhook.error_codes import WebhookErrorCode
+from saleor.webhook.event_types import WebhookEventAsyncType
+
+
+class NotifyUserEventValidationMixin:
+    @classmethod
+    def validate_events(cls, events):
+        # NOTIFY_USER needs to be the only one event registered per webhook.
+        # This solves issue temporarily. NOTIFY_USER will be deprecated in the future.
+        if WebhookEventAsyncType.NOTIFY_USER in events and len(events) > 1:
+            raise ValidationError(
+                {
+                    "async_events": ValidationError(
+                        "Notify User event has to be the only one event per webhook.",
```

Comment:
I would rephrase a bit to something like this:
"The `NOTIFY_USER` webhook cannot be combined with other events."